assets/app/view/game/round/stock.rb

```diff
@@ -375,6 +376,46 @@ def render_bid_input(company)
 
           [h(Bid, entity: @current_entity, corporation: company)]
         end
+
+        def render_buyable_minors
+          props = {
+            style: {
+              display: 'inline-block',
+              verticalAlign: 'top',
+            },
+          }
+
+          @step.buyable_minors.map do |minor|
+            children = [h(Corporation, corporation: minor)]
+            children << render_minor_input(minor) if @selected_corporation == minor
+            h(:div, props, children)
+          end
+        end
+
+        def render_minor_input(minor)
+          minor_actions = []
```

Comment:
minor_actions[render_minor_buy]

Reply:
why do you do concat?

can't you just do div, render_minor_buy?

Reply:
Remains from copy paste...

Have improved now.